stdlib/public/core/ObjectIdentifier.swift

```diff
@@ -65,6 +65,11 @@ public struct ObjectIdentifier: Sendable {
   public init(_ x: Any.Type) {
     self._value = unsafeBitCast(x, to: Builtin.RawPointer.self)
   }
+
+  @_alwaysEmitIntoClient
+  public init(_ x: any (~Copyable & ~Escapable).Type) {
```

Comment:
You could just convert the old `Any.Type` initializer and mark it `@_preInverseGenerics`.

Reply:
Oh! I thought `Any.Type` wouldn't ever include noncopyable/nonescapable stuff. I'll put it on the to do list! 👍